### ISSUE TITLE
Question about the disabling gradients operations

### ISSUE THREAD

**Issue by yoyoyoohh** [TaoHuang2018/Neighbor2Neighbor#3]
Hi, I am confused about the code in the training_cript.md:
```
with torch.no_grad():
    noisy_denoised = network(noisy)
```
It seems that the disabling gradients operation didn't appear in the paper, and could you please explain the special purpose for doing this?

**Comment by FunkyKoki** [comment]
Check the paper, for the noisy image with the original size (not sub-sampled), it also needs to go through the Denoising Network and yields the regularizer term.

**Comment by yoyoyoohh** [comment]
> Check the paper, for the noisy image with the original size (not sub-sampled), it also needs to go through the Denoising Network and yields the regularizer term.

But why should it stop the gradients produced by the original size image?

**Comment by TaoHuang2018** [comment]
It is an empirical training technique. Stopping the gradient is helpful to stabilize the training. It does not violate the theoretical motivation.

**Comment by unpairdenosie** [comment]
I'm a little confused about the question. We don't have the groundtruth of the original size image in this paper, so how to get the gradients produced by the original size image?
